packages/server/lib/util/ci_provider.js

```diff
@@ -465,6 +467,7 @@ const _providerCommitParams = () => {
     azure: {
       sha: env.BUILD_SOURCEVERSION,
       branch: env.BUILD_SOURCEBRANCHNAME,
+      defaultBranch: env.SYSYSTEM_PULLREQUEST_TARGETBRANCH,
```

Comment:
This would be the target branch for the current pull request, which is not necessarily the DEFAULT branch of the repo. That's how I read it, but maybe you checked it individually. These things are always confusing.

Reply:
I did not setup an Azure pipeline to verify. Believe this is correct per the [docs](https://learn.microsoft.com/en-us/azure/devops/repos/git/pull-requests?view=azure-devops&tabs=browser#change-the-target-branch-of-an-active-pull-request) which is using the term `target` instead of `base`:

> For most teams, nearly all PRs target a default branch, such as main or develop

Reply:
@emilyrohrbough They have a 'default branch' setting. which would be set for the repo. That would be different from the target branch for a PR. I'm not sure it's provided as an env var though. https://learn.microsoft.com/en-us/azure/devops/repos/git/manage-your-branches?view=azure-devops#change-your-default-branch

Reply:
recommendation: just collect as many variants to cover all cases

Reply:
You can collect what you like, but this env var is attributing it as the default branch, which could cause innaccuracies with the ways we already use this value if it’s incorrect.

Reply:
@jennifer-shehane good callout. That was my mistake with the mappings here. I have updated and also added missing envs for a few other providers.